package-res/reportviewer/reportviewer.js

```diff
@@ -934,6 +934,8 @@ define([ 'common-ui/util/util', 'common-ui/util/timeutil', 'common-ui/util/forma
         // that the report content is being updated.
         me.reportPrompt.showGlassPane();
 
+        me._setParametersDisablement(true);
```

Comment:
Can you rename that function to "updateParametersDisabledState(..)" 

you are not setting a single property but manipulate a whole bunch of elements, so "set" is a misleading prefix. And I would argue that Disablement is not quite a suitable description for what you are doing (updating the disabled state).